packages/react-components/src/icons/src/index.ts

```diff
@@ -6,14 +7,14 @@ export * from "./MultiVariantIcon";
 export * from "./IconList";
 export * from "./EmbeddedIcon";
 
-function createOrbitIcon(source, displayName) {
+function createOrbitIcon(source: ElementType, displayName: string): ReturnType<typeof Icon.create> {
     const Component = Icon.create(source);
     Component.displayName = displayName;
 
     return Component;
 }
 
-function createOrbitMultiVariantIcon(source24, source32, displayName) {
+function createOrbitMultiVariantIcon(source24: ElementType, source32: ElementType, displayName: string): ReturnType<typeof MultiVariantIcon.create> {
```

Comment:
i prefer the `ReturnType<typeof MultiVariantIcon.create>` synthax over `OrbitComponent<"svg", CreatedMultiVariantIconProps>`. What do you think?